pages/common/ping6.md

```diff
@@ -0,0 +1,23 @@
+# ping6
+
+> Send ICMP ECHO_REQUEST packets to network hosts via IPv6 address.
+
+- Ping host:
```

Comment:
Could you change this to `Ping a host:` for consistency. Same with the other ones. 👍

Reply:
btw, this issue is there in the original ping.md page too. We should fix that.